src/API/app.ts

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
* Folder `src` is for the client app only; the API should not be included
* I'd rather have a lowercase folder name

Reply:
Same. this looks great otherwise.